app/routes/endemics/date-of-visit.js

```diff
@@ -164,13 +164,31 @@ module.exports = [
           request.payload[labels.month] - 1,
           request.payload[labels.day]
         )
-        const { isValid, content } = typeOfReview === claimType.review ? isValidReviewDate(previousClaims, dateOfVisit) : isValidEndemicsDate(previousClaims, dateOfVisit, organisation?.name, formattedTypeOfLivestock)
+        const { isValid, reason } = isValidDateOfVisit(dateOfVisit, typeOfReview, previousClaims, latestVetVisitApplication)
+        const content = {}
         if (!isValid) {
+          switch (reason) {
+            case dateOfVetVisitExceptions.reviewWithin10:
+              content.url = '#'
```

Comment:
Move this outside case as its not changing in any case.

Reply:
It will change once we have real urls, this is just a placeholder for now